docs/features/containers.md

```diff
@@ -537,6 +537,26 @@ const { output, exitCode } = await container.exec(["echo", "hello", "world"], {
 	}
 });
 ```
+To handle cases where you need separate outputs for `stdout` and `stderr`, you can use the `execVerbose` method. 
```

Comment:
Instead of adding a 2nd `exec` command, would it be possible to add the `stdout` and `stderr` fields to the existing `exec` response? It's a lot of code being added for a slight change

Reply:
I initially attempted the approach you suggested. However, adopting this method would result in the ExecResult type being defined as either:

```typescript
type ExecResult = { output: string; exitCode: number } | { stdout: string; stderr: string; exitCode: number };
```
or:

```typescript
type ExecResult = { output?: string; stdout?: string; stderr?: string; exitCode: number };
```
Both options are technically feasible, but they introduce potential confusion and require modifications to the existing test code that uses the exec functionality. This is because the current test cases assume that the output field is always present. Changing the type as proposed makes it challenging for the TypeScript compiler to validate whether output exists or not.

If you're okay with proceeding despite these issues, I can move forward with your suggestion and update the relevant test cases accordingly. Please let me know your preference.

Reply:
Understood, I also prefer the separate `exec` method instead of that return type. Is it not possible to have them all?

```ts
type ExecResult = { output: string; stdout: string; stderr: string; exitCode: number };
```

Reply:
Understood. I have rewritten the exec code to always return an ExecResult with the following type:

```typescript
type ExecResult = { output: string; stdout: string; stderr: string; exitCode: number };
```
Additionally, I have added a few test cases for this implementation. Please review them and let me know your thoughts.